packages/components/src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -47,6 +57,8 @@ export type DescriptionListProps = BaseComponentPropsWithChildren<
   {
     /**Påvirker tekst styling. */
     appearance?: DescriptionListAppearance;
+    /**Setter flex-direction. Default er column. */
```

Comment:
Bruk gjerne JSDoc annotasjoner
```suggestion
    /**Setter flex-direction. 
    * @deafult 'column'
    */
```

Reply:
Fixed